API.md

```diff
@@ -503,36 +783,132 @@ frisby.create('First test')
 ```
 
 ### only()
+
 Exclusively run this test. When `toss()` is invoked, the test is wrapped in a
 Mocha `describe.only` block instead of a `describe` block.
 
+### timeout(ms)
+
+Sets the timeout for this request.
+
+| Parameter | Description | Required |
+| --------- | ----------- | -------- |
+| ms | Integer. Sets the mocha timeout for this request. | Yes |
+
+```javascript
+frisby.create('Long-running request')
+    .get('http://example.com/slowthing')
+    .timeout(5000)
+    .expectStatus(200)
+    .toss()
+```
+
+This function can also be called with no parameter to return the current configured timeout (either by default, by global setup or having used this function with a parameter previously).
+
+This is different from [expectMaxResponseTime](#expectmaxresponsetimems) in that expectMaxResponseTime will await the request to complete then test the amount of time that the request took (after any preceeding expect functions). This will abort the request and so not run any developer-defined expect functions on the response.
```

Comment:
> This will abort the request and so not run any developer-defined expect functions on the response.

This is not guaranteed. At least I think it's not guaranteed.

How about this?

> When a timeout occurs, the test will be aborted. The expectations and inspections may or may not run and will not be printed. If you want to run all the expectations and inspections even when the response is slow, use `expectMaxResponseTime` instead.

Reply:
I think I got that nugget from the comments on the method, but your version is safer and snappier.

Note to self: Test this.